### DISCUSSION TITLE
[替代notion] 换成国内类似 notion的平替：妙记多

### DISCUSSION THREAD

**Discussion by shangkouyou** [tangly1024/NotionNext#2076] [Ideas]
**为什么提出这个新的特性改动**
notion 大部份人还是有一定困难 国内仿的很多 

**描述一下你推荐的解决方案**
期待大佬的看上去动力

**描述一下你考虑过的其它替代解决方案**
flowus 息流。 Mojidoc 妙记多

**补充说明**
替代方案那边填写的 国内的类似notion的 要死也可以就

**Comment by expoli** [comment]
暂时不可以，以前讨论过，没有相应的 API 和 SDK 无法兼容语雀等

**Comment by shangkouyou** [comment]
https://www.mojidoc.com/
大佬可以看下这个吗 跟notion一样
![image](https://github.com/tangly1024/NotionNext/assets/73685156/ef6f9921-42c7-4c72-8506-bbe8116ef2b5)

**Comment by shangkouyou** [comment]
> 暂时不可以，以前讨论过，没有相应的 API 和 SDK 无法兼容语雀等
Mojidoc妙记多 跟notion差不多是一样的 这个是国内的公司在做的 大佬要不要看下把这个也渲染成博客呀

**Comment by dreamer2q** [comment]
...

国内的玩意，想多了

yuque 就是一个例子

**Comment by igstranger** [comment]
如果换成这玩意我宁愿不用
